src/components/Atoms/Link/Link.js

```diff
@@ -27,9 +24,19 @@ const Link = ({
    * this is an internal link OR on our whitelist before making it a '_self' link
    */
   if (target === null) {
-    const isExternalLink = domainRegEx.test(href);
```

Comment:
I've removed this here as the same functionality is replaced (and improved) by the new code